src/Config.php

```diff
@@ -3415,6 +3427,42 @@ public function showFormSecurity()
         echo '</td>';
         echo '</tr>';
 
+        echo '<tr class="tab_bg_1">';
```

Comment:
as we target main, we may take advantage to move to twig template for this whole security form